tests/test_sockets.py

```diff
@@ -483,9 +527,9 @@ async def test_extra_attributes(self, server_sock, socket_path):
             pytest.raises(TypedAttributeLookupError, stream.extra, SocketAttribute.local_port)
             pytest.raises(TypedAttributeLookupError, stream.extra, SocketAttribute.remote_port)
 
-    @pytest.mark.parametrize('as_path', [False, True], ids=['str', 'path'])
-    async def test_send_receive(self, server_sock, socket_path, as_path):
```

Comment:
this as_path was unused

Reply:
Then perhaps we should add this?
```python3
        if not as_path:
            socket_path = str(socket_path)
```

Reply:
I fixed it at runtime by making a Union[str, Path] fixture

Reply:
You can't reassign a Path to a str, in refactoring it to a fixture I noticed the bug in the test